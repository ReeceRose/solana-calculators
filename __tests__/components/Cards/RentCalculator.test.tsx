/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/react';

import RentCalculatorCard from '../../../components/Cards/RentCalculator';

test('RentCalculator card component matches snapshot', () => {
  const { asFragment } = render(<RentCalculatorCard />);
  expect(asFragment()).toMatchSnapshot();
});
