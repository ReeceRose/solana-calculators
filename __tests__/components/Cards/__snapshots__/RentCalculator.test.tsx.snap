// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RentCalculator card component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="relative flex flex-col w-full min-w-0 mb-6 break-words border-0 rounded-lg shadow-lg dark:bg-gray-700"
  >
    <div
      class="px-6 py-6 mb-0 bg-white rounded-t dark:bg-gray-900"
    >
      <div
        class="flex justify-between text-center"
      >
        <h6
          class="text-xl font-bold text-blue-500"
        >
          Rent Calculator
        </h6>
        <button
          class="px-4 py-2 mr-1 text-xs font-bold text-blue-500 uppercase transition-all duration-150 ease-linear bg-gray-100 rounded shadow outline-none dark:text-white dark:bg-gray-500 active:bg-gray-600 hover:shadow-md focus:outline-none"
          type="button"
        >
          Calculate
        </button>
      </div>
    </div>
    <div
      class="flex-auto px-4 lg:px-10"
    >
      <hr
        class="my-6 border-gray-800 dark:border-gray-50 border-b-1"
      />
      <h6
        class="text-sm font-bold text-gray-600 uppercase dark:text-white"
      >
        Current Price per byte-year: 
        <span
          class="font-extrabold text-gray-700 dark:text-blue-500"
        >
          0.00000348 SOL
        </span>
      </h6>
    </div>
    <div
      class="flex-auto px-4 lg:px-10"
    >
      <form>
        <hr
          class="my-6 border-gray-800 dark:border-gray-50 border-b-1"
        />
        <h6
          class="mt-3 mb-6 text-sm font-bold text-gray-600 uppercase dark:text-white"
        >
          Calculator
        </h6>
        <div
          class="flex flex-wrap"
        >
          <div
            class="w-full px-4 lg:w-6/12"
          >
            <div
              class="relative w-full mb-3"
            >
              <label
                class="block mb-2 text-xs font-bold text-gray-600 uppercase dark:text-white"
                for="kilobytes"
              >
                Kilobytes
              </label>
              <input
                class="w-full px-3 py-3 text-sm text-gray-600 placeholder-gray-300 transition-all duration-150 ease-linear bg-white border-0 rounded shadow focus:outline-none focus:ring"
                type="number"
                value="128"
              />
            </div>
          </div>
        </div>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
